nextflow.config

```diff
@@ -37,6 +38,7 @@ params {
   help = false
   show_supported_models = false
   tracedir = "${params.outdir}/pipeline_info"
+  igenomes_base = 's3://ngi-igenomes/igenomes/'
```

Comment:
```suggestion
```
it is not used in this pipeline, right?

Reply:
I remembered that we also kept this in the `hlatyping` pipeline. Can't remember why exactly but I think it doesn't hurt since there is `igenome_ignore`